gradle/dependencies.gradle

```diff
@@ -60,8 +60,6 @@ versions += [
   apacheda: "1.0.2",
   apacheds: "2.0.0-M24",
   argparse4j: "0.7.0",
-  avro: "1.9.1",
```

Comment:
Does this need to be taken out of `confluent-import-control.xml` too?

Reply:
Good point, we can. remove that file. Done.